### PR TITLE
Add sort file by date functionality to Browser

### PR DESCRIPTION
**Pull request by alelong-work**

### CODE REVIEW

**Comment by bdukes** [comment]
## Before
File browser dialog without date sorting



## After
File browser dialog with date sorting

File: Browser/Browser.aspx.cs [hunk 1]
```diff
@@ -2067,6 +2202,18 @@ private void SetLanguage()
                  Localization.GetString("SortDescending.Help", ResXFile, LanguageCode));
             SortDescending.ToolTip = Localization.GetString("SortDescending.Help", ResXFile, LanguageCode);
 
+            SortByDateAscending.Text = string.Format(
+                 "<img src=\"Images/AscendingArrow.png\" alt=\"{0}\" title=\"{1}\" />Date",
```
Comment (bdukes) [inline Browser/Browser.aspx.cs]: @along-engage this _Date_ text also needs to come from a resource file

**Review by bdukes — CHANGES_REQUESTED** [review]

File: Browser/Browser.aspx.cs (hunk 1 above)
Comment (bdukes) [reply Browser/Browser.aspx.cs]: @along-engage you got the second one, still need to fix this one.  Thanks!

**Review by bdukes — CHANGES_REQUESTED** [review]

File: Browser/Browser.aspx.cs (hunk 1 above)
Comment (alelong-work) [reply Browser/Browser.aspx.cs]: @bdukes My mistake, this has been changed as well!

File: Browser/Browser.aspx.cs [hunk 2]
```diff
@@ -2067,6 +2202,20 @@ private void SetLanguage()
                  Localization.GetString("SortDescending.Help", ResXFile, LanguageCode));
             SortDescending.ToolTip = Localization.GetString("SortDescending.Help", ResXFile, LanguageCode);
 
+            SortByDateAscending.Text = string.Format(
+                 "<img src=\"Images/AscendingArrow.png\" alt=\"{0}\" title=\"{1}\" />{2}",
+                 Localization.GetString("SortByDateAscending.Text", ResXFile, LanguageCode),
+                 Localization.GetString("SortByDateAscending.Help", ResXFile, LanguageCode));
+            Localization.GetString("SortByDate.Text", ResXFile, LanguageCode));
```
Comment (bdukes) [inline Browser/Browser.aspx.cs]:
```suggestion
                 Localization.GetString("SortByDateAscending.Help", ResXFile, LanguageCode),
                 Localization.GetString("SortByDate.Text", ResXFile, LanguageCode));
```

**Review by bdukes — CHANGES_REQUESTED** [review]

**Review by bdukes — APPROVED** [review]

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**